### PR TITLE
Removed the experimental labels on the delete tenant API.

### PR DESCRIPTION
**Pull request by 09jvilla**
Updated the docs to remove the experimental labels on the delete tenant API as we are promoting this to stable.

### CODE REVIEW

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 1]
```diff
@@ -1021,6 +1021,6 @@ Requires [authentication](#authentication).
 GET /compactor/delete_tenant_status
 ```
 
-Returns status of tenant deletion. Output format to be defined. Experimental.
+Returns status of tenant deletion.
```
Comment (09jvilla) [inline docs/sources/operators-guide/reference-http-api/index.md:1024]: I just deleted "Output format to be defined." Not sure if we know the output format and then I could update these docs to include that.

**Review by pracucci — COMMENTED** [review]
Could you also remove the `- Tenant deletion API` entry from `docs/sources/operators-guide/configure/about-versioning.md`, and announce these API to be stable in the `CHANGELOG`, please?

File: docs/sources/operators-guide/reference-http-api/index.md (hunk 1 above)
Comment (pracucci) [reply docs/sources/operators-guide/reference-http-api/index.md:1024]: The response is a JSON like this:

```json
{
  "tenant_id": "<id>",
  "blocks_deleted": true
}
```

Where `blocks_deleted` is `true` if all the tenant's blocks have been deleted.

Comment (09jvilla) [reply docs/sources/operators-guide/reference-http-api/index.md:1024]: added!

**Comment by 09jvilla** [comment]
Updated to include @pracucci 's feedback.

File: docs/sources/operators-guide/reference-http-api/index.md [hunk 2]
```diff
@@ -1021,6 +1021,16 @@ Requires [authentication](#authentication).
...
+
+#### Response schema
+
+```json
+{
+  "tenant_id": "<id>",
+  "blocks_deleted": true
+}
+```
+The `blocks_deleted` field will be set to `true` if all the tenant's blocks have been deleted.
```
Comment (pstibrany) [inline docs/sources/operators-guide/reference-http-api/index.md:1034]: Linter requires extra newline between end of code block and the text.

Comment (pstibrany) [reply docs/sources/operators-guide/reference-http-api/index.md]: Added directly to your branch.

**Review by pstibrany — APPROVED** [review]

**Pull request merged**